### ISSUE TITLE
[Feature Request]: Show comments on songs/videos

### ISSUE THREAD

**Issue by change-agent** [th-ch/youtube-music#2372]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

It would be great if we could view and interact with comments on songs/videos, like we can do [on Android and iOS](https://9to5google.com/2023/09/01/youtube-music-now-playing-redesign-2023-comments/).

### Proposed Solution

Comments tab here: 
![CleanShot 2024-08-28 at 08 56 06@2x](https://github.com/user-attachments/assets/8873972c-93df-408b-a2a6-90c23b6e8dec)

### Alternatives Considered

N/A

**Comment by JellyBrick** [comment]
Unfortunately, Desktop version of YTM doesn't provide data of the comments.

**Comment by change-agent** [comment]
@JellyBrick if you remove the "music" subdomain from a URL, you can see the comments. They are just YouTube comments on the YouTube video equivalent of the track – e.g. https://music.youtube.com/watch?v=jTmX9xW1oHY&si=dzIA1PvkrHqJxmrp 
vs https://www.youtube.com/watch?v=jTmX9xW1oHY:

![CleanShot 2024-08-30 at 08 52 35@2x](https://github.com/user-attachments/assets/2070d0a8-da51-4a1c-8df4-9650493edc77)

![CleanShot 2024-08-30 at 08 53 09@2x](https://github.com/user-attachments/assets/30659e38-9990-4467-a44a-d7c57771f72e)

**Comment by change-agent** [comment]
@th-ch Keen to hear your thoughts on the above

**Comment by ArjixWasTaken** [comment]
> @JellyBrick if you remove the "music" subdomain from a URL, you can see the comments. They are just YouTube comments on the YouTube video equivalent of the track – e.g. https://music.youtube.com/watch?v=jTmX9xW1oHY _vs_ https://www.youtube.com/watch?v=jTmX9xW1oHY:
> On that note, it would be great if you could surface the number of "likes" on a song too, since that's also data sourced from the YouTube backend.

the `music.` prefix is irrelevant to implementing this feature
we'd have to use YouTube's API via [`youtubei.js`](https://github.com/LuanRT/YouTube.js)

which I must admit is quite easy to do, but it has the risk of Google flagging your IP as suspicious

**Comment by nowhereman87** [comment]
Agree. Being able to read the comments while listening to the music would be great, I think it's one of the things that often makes me opt for YT instead of Spotify or Tidal

**Comment by Yocracra** [comment]
> the `music.` prefix is irrelevant to implementing this feature we'd have to use YouTube's API via [`youtubei.js`](https://github.com/LuanRT/YouTube.js)
> 
> which I must admit is quite easy to do, but it has the risk of Google flagging your IP as suspicious

What's the risk behind this? Having to re-login sometimes? YTM refusing to work?

I use Mullvad VPN just about 100% of the time on all my devices, and I haven't had any problems with YTM. Since Mullvad VPN is stupidly secure, Mullvad's IPs are suuuper suspicious because people like to use them for illegal things. I have to do extra captchas from Google all the time, and some sites straight-up will not let me in until I turn it off.

But, again, no problems from YTM.

**Comment by ArjixWasTaken** [comment]
I am not sure, but I assume comments are monitored more due to bots.

Maybe we could use Google's cloud console API for YouTube and have users login so we can use the API.

**Comment by SBagelton** [comment]
> > [@JellyBrick](https://github.com/JellyBrick) if you remove the "music" subdomain from a URL, you can see the comments. They are just YouTube comments on the YouTube video equivalent of the track – e.g. https://music.youtube.com/watch?v=jTmX9xW1oHY _vs_ https://www.youtube.com/watch?v=jTmX9xW1oHY:
> > On that note, it would be great if you could surface the number of "likes" on a song too, since that's also data sourced from the YouTube backend.
> 
> the `music.` prefix is irrelevant to implementing this feature we'd have to use YouTube's API via [`youtubei.js`](https://github.com/LuanRT/YouTube.js)
> 
> which I must admit is quite easy to do, but it has the risk of Google flagging your IP as suspicious

Is this the thing where it blocks your ip from making youtube api requests for 24 hours?

**Comment by javalosv** [comment]
Much appreciate execute some test to validate any restriction. This feacture will allow high interation on logging users.
